src/main/window-lifecycle.js

```diff
@@ -84,8 +83,6 @@ export async function createMainWindow({ dimensions, positions }: any) {
     mainWindow.webContents.openDevTools();
   }
 
-  mainWindow.on("close", terminate);
```

Comment:
how is this handle now? does quitting the app still works without this?

Reply:
When a window is closed and there are no more active windows, electron quit the app gracefully

Reply:
actually, it is not electron's default behavior, we have an event waiting for all windows to be closed